src/main/java/net/fabricmc/loom/util/ModPlatform.java

```diff
@@ -48,21 +53,50 @@ public boolean isExperimental() {
 		return experimental;
 	}
 
-	public static void assertPlatform(Project project, ModPlatform platform) {
-		assertPlatform(LoomGradleExtension.get(project), platform);
+	public static void assertPlatform(Project project, ModPlatform... platforms) {
+		assertPlatform(LoomGradleExtension.get(project), platforms);
 	}
 
-	public static void assertPlatform(LoomGradleExtensionAPI extension, ModPlatform platform) {
-		assertPlatform(extension, platform, () -> {
-			String msg = "Loom is not running on %s.%nYou can switch to it by adding 'loom.platform = %s' to your gradle.properties";
-			String name = platform.name().toLowerCase(Locale.ROOT);
-			return msg.formatted(name, name);
-		});
+	public static void assertPlatform(LoomGradleExtensionAPI extension, ModPlatform... platforms) {
+		assertPlatform(extension, () -> {
+			String msg = "Loom is not running on any of %s.%nYou can switch to it by any of the following: Add any of %s to your gradle.properties";
```

Comment:
Nit: This should not say "any of [FORGE]" for single-platform cases. Probably best to just check if `platforms.length == 1` and use the old message format.